### ISSUE TITLE
Incorrect transpiling of export statement 

### ISSUE THREAD

**Issue by starsolaris** [fuse-box/fuse-box#1945]
fuse-box: 4.0.0-next.440

```
export const showEventInfo = createAction(
  'SHOW_EVENT_INFO',
  action => (showEventInfo: boolean) => action(showEventInfo)
);
```

will produce:
```
exports.showEventInfo = createAction('SHOW_EVENT_INFO', action => showEventInfo => action(exports.showEventInfo));
```

instead of:
```
exports.showEventInfo = createAction('SHOW_EVENT_INFO', action => showEventInfo => action(showEventInfo));
```

**Comment by starsolaris** [comment]
maybe this is duplicate of #1942

**Comment by nchanged** [comment]
hm...  I will find some time during the weekend for these bugs.

**Comment by nchanged** [comment]
@starsolaris please verify the fix (both issues) in 4.0.0-next.441

**Comment by starsolaris** [comment]
@nchanged, works as expected. 👍
